Makefile

```diff
@@ -6,10 +6,10 @@ DHRY-CFLAGS += -fno-builtin-printf -fno-common -falign-functions=4
 #Uncomment below for FPGA run, default DHRY_ITERS is 2000 for RTL
 #DHRY-CFLAGS += -DDHRY_ITERS=20000000
 
-SRC = dhry_1.c dhry_2.c
+SRC = dhry_1.c dhry_2.c strcmp.S
 HDR = dhry.h
 
-override CFLAGS += $(DHRY-CFLAGS) $(XCFLAGS) -Xlinker --defsym=__stack_size=0x800 -Xlinker --defsym=__heap_size=0x1000
+override CFLAGS += $(DHRY-CFLAGS) $(XCFLAGS) -mtune=sifive-7-series -Xlinker --defsym=__stack_size=0x800 -Xlinker --defsym=__heap_size=0x1000
```

Comment:
CFLAGS changes might open another PR?

Reply:
Oh...sorry. this should not be part of this patch.
I will remove it.